tests/test_cases/test_3316_c/check_version.sh

```diff
@@ -0,0 +1,25 @@
+# Copyright cocotb contributors
```

Comment:
I might have implemented this with e. g. Python's packaging module since we will always comply to PEP440, but I guess this is fine too.

Reply:
Actually, maybe leave it as is but rename the file so that it doesn't have the .sh extension, so that we could reimplement without other changes?